### PR TITLE
Fix error: When the injector is not the first of wells, opm should work well

### PR DESCRIPTION
**Pull request by qilicun**
This PR fix the following error: if wells object is not start with a injector, current flow_polymer will throw. 
After this fix, flow_polymer should work as expected.

### CODE REVIEW

File: opm/polymer/PolymerInflow.cpp [hunk 1]
```diff
@@ -165,12 +165,13 @@ namespace Opm
...
                 map_it = wellPolymerRate_.find(wells.name[wix]);
-                if (map_it == wellPolymerRate_.end()) {
-                    OPM_THROW(std::runtime_error, "Could not find a match for well from WPOLYMER.");
-                } else {
+                if (map_it != wellPolymerRate_.end()) {
                     break;
                 }
             }
+            if (wix == wells.number_of_wells) {
+                OPM_THROW(std::runtime_error, "Could not find a match for well from WPOLYMER.");
```
Comment (atgeirr) [inline opm/polymer/PolymerInflow.cpp]: I suggest adding the name of the well to the message:
```
OPM_THROW(std::runtime_error, "Could not find a match for well " << wells.name[wix] << " from WPOLYMER.");
```

Comment (qilicun) [inline opm/polymer/PolymerInflow.cpp]: Good point, I will update the PR.

**Comment by GitPaean** [comment]
Can you tell under what kind of situation that `OPM_THROW(std::runtime_error ..` will happen?

**Comment by qilicun** [comment]
For example, the following situation will cause the problem, which put the producer first.
```
WELSPECS
--     WELL   GROUP LOCATION     BHP   PI
--     NAME   NAME     I    J   DEPTH DEFN
    'PROD' 'G'  400  1    1000 'OIL'  /
    'INJE' 'G'  1  1    1000 'WATER'  /
/
```

**Comment by GitPaean** [comment]
No, I mean after this PR. I am aware what this PR is trying to fix and tested it already. 

After this PR, under what kind of situation, `OPM_THROW(std::runtime_error ..` can be triggered.

**Comment by atgeirr** [comment]
It would be a situation like
```
WPOLYMER
 INJ1   0.3   1.0   /
/
```
when there is no well named `INJ1`.

**Comment by GitPaean** [comment]
I think parser will cause problem there already. Anyway, it is not something very important here.

**Pull request merged**